src/lualib/tsconfig.lua50.json

```diff
@@ -7,14 +7,13 @@
         "skipLibCheck": true,
         "rootDirs": [".", "5.0"],
 
-        "noUnusedLocals": true,
-        "noUnusedParameters": true
+        "strict": true
     },
     "tstl": {
         "luaLibImport": "none",
         "noHeader": true,
         "luaTarget": "5.0",
         "luaPlugins": [{ "name": "../../dist/lualib-build/plugin.js" }]
     },
-    "include": ["*.ts", "5.0/*.ts", "declarations", "../../language-extensions/index.d.ts"]
+    "include": ["*.ts", "5.0/**/*.ts", "declarations/**/*.ts", "../../language-extensions/index.d.ts"]
```

Comment:
can you simplify this to `"./**/.*.ts"`

Reply:
no because I expicitly want ot include 5.0 and not universal